src/tests/ftest/util/soak_utils.py

```diff
@@ -490,9 +491,12 @@ def launch_reboot(self, pools, name, results, args):
     # If all ranks "joined", issue reintegrate for all pool on all ranks and wait for
     #    rebuild to complete
     # Update multiprocessing queue with results and args
+    # pylint: disable=too-many-nested-blocks
```

Comment:
Should also ignore this check because pylint is failing. Ideally we should fix it properly but that would require refactoring this function
```suggestion
    # pylint: disable=too-many-nested-blocks,too-many-branches
```